tracks/ruby/exercises/acronym/readme.md

```diff
@@ -0,0 +1,44 @@
+# Acronym
+
+_Acronym_ is one of the first side exercises, disclosed by _TwoFer_
+
+## Reasonable Solutions
+
+```ruby
+module Acronym
+  def self.abbreviate(fullname)
+    fullname.scan(/\b[a-zA-Z]).join.upcase
```

Comment:
Maybe talk about naming the regular expressions by holding them in a dictionary of some kind, or even directly as a variable.

Reply:
Cool, I'll add that as a talking point. Thanks.